### PR TITLE
Added Login and Register

### PR DESCRIPTION
**Pull request by aarontorres0**

### CODE REVIEW

**Review by jlipscomb071 — APPROVED** [review]
LGTM

File: firebaseConfig.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  apiKey: process.env.FIREBASE_API_KEY,
+  authDomain: process.env.FIREBASE_AUTH_DOMAIN,
+  projectId: process.env.FIREBASE_PROJECT_ID,
+  storageBucket: process.env.FIREBASE_STORAGE_BUCKET,
+  messagingSenderId: process.env.FIREBASE_MESSAGING_SENDER_ID,
+  appId: process.env.FIREBASE_APP_ID,
+};
+
+/*
+const firebaseConfig = {
```
Comment (jlipscomb071) [inline firebaseConfig.js]: Remove this

File: metro.config.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+// Learn more https://docs.expo.io/guides/customizing-metro
```
Comment (jlipscomb071) [inline metro.config.js:1]: Remove this comment.

File: firebaseConfig.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  storageBucket: "abc.appspot.com",
+  messagingSenderId: "abc",
+  appId: "abc",
+};
+*/
+
+export const FIREBASE_APP = initializeApp(firebaseConfig);
+export const FIRESTORE_DB = getFirestore(FIREBASE_APP);
+export const FIREBASE_AUTH = getAuth(FIREBASE_APP);
+// For more information on how to access Firebase in your project,
```
Comment (jlipscomb071) [inline firebaseConfig.js]: Remove these comments.

**Review by jlipscomb071 — COMMENTED** [review]
Added a few comments. Just need to remove some syntax comments. Other than that, lgtm.

**Comment by oosinloye** [comment]
LGTM!

**Review by oosinloye — APPROVED** [review]

**Pull request merged**